Graphical Backup Program/Form.cs

```diff
@@ -297,16 +295,20 @@ private static bool ValidGroupChar(char group)
         /// <param name="url">The URL to attempt at opening</param>
         private static void OpenUrl(string url)
         {
-            //TODO: Better handle any exceptions
-            try
-            {
-                Process.Start(url);
-            }
-            catch
-            {
-                url = url.Replace("&", "^&");
-                Process.Start(new ProcessStartInfo("cmd", $"/c start {url}") { CreateNoWindow = true });
-            }
+			url = url.Replace("&", "^&"); //This is needed because command prompt needs to know that an ampersand in the url var is not to be parsed as a second command; it is part of the url
+
+			if (url.StartsWith("http://"))
+			{
+                MessageBox.Show("The URL is not going to a secure version of the website, please double check the website loads as HTTPS", "SSL Site Not Requested", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+			}
+
+			else if (!url.StartsWith("https://"))
+			{
+                url = $"https://{url}";
+			}
+
+            //TODO: Possible malicious code execution point? are we parsing url well enough to prevent naughty commands?
```

Comment:
Good point! Never thought of that

Reply:
I really don't believe this would be a "popular" vector of attack, but figured any code should always be aware of any risk it poses to the system.